plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/GoogleLoginCommandHandler.java

```diff
@@ -21,19 +21,14 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
     Credential credential = loginService.getCachedActiveCredential();
     if (credential == null) {
-      credential = loginService.getActiveCredential();
+      loginService.getActiveCredential();
```

Comment:
I wonder if getActiveCredential might be better named loadCredential or just login. Generally to me a getFoo method suggests that the method merely returns a value, not performs an action.